Tests/Testably.Abstractions.Tests/FileSystem/Directory/DeleteTests.cs

```diff
@@ -135,10 +135,17 @@ public void
 	[SkippableTheory]
 	[AutoData]
 	public void Delete_Recursive_WithSubdirectory_ShouldDeleteDirectoryWithContent(
-		string path, string subdirectory)
+		string path, string subdirectory, string fileName, string fileContent)
```

Comment:
I would add this as a separate test `Delete_Recursive_WithFileInSubdirectory_ShouldDeleteDirectoryWithContent`.